src/main/java/nextstep/qna/domain/Answer.java

```diff
@@ -72,6 +77,14 @@ public void toQuestion(Question question) {
         this.question = question;
     }
 
+    public void delete() {
+        setDeleted(true);
```

Comment:
```suggestion
        this.deleted = true;
```
setter 메서드를 제거하면 어떨까?